gammapy/maps/core.py

```diff
@@ -776,11 +780,16 @@ def interp_to_geom(self, geom, **kwargs):
         if self.data.dtype == bool:
             kwargs.setdefault("interp", 0)
 
-        data = self.interp_by_coord(coords, **kwargs)
+        if preserve_counts:
+            old_map_copy = self.copy()
+            old_map_copy.data /= self.geom.solid_angle().to_value("deg2")
```

Comment:
I just realised, that the bins size in energy could change as well. So we probably need a more general solution like:
```python3
bin_volume = self.geom.bin_volume.value
old_map_copy.data /= bin_volume

data *= geom.bin_volume.to_value(bin_volume.unit)

```

Reply:
Conclusion from the dev meeting today: keep the implementation, but add a check on the equality of the energy axis. This simply works like `energy_axis_1 == energy_axis_2`...

Reply:
done! :+1: